src/nih_wayfinding/app/styles/components/_type.scss

```diff
@@ -0,0 +1,5 @@
+h1, h2, h3, h4, h5, h6 {
+  &:first-child {
```

Comment:
I find this really helpful in applications, where you generally just want your headings at the top of the element and handle spacing by adding padding to the element.